src/components/Button/Button.css

```
@@ -0,0 +1,103 @@
+*{
+    @import url('https://fonts.googleapis.com/css2?family=Mulish:wght@300;400;700&display=swap');
+}
+.App{
+    display: flex;
+    flex-direction: row;
+}
+.App button{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    font-family: 'Mulish', sans-serif;
+    font-size: 18px;
+    font-weight: 600;
+    line-height: 24px;
+    gap:10px;
+    height: 56px;
+    padding: 16px 32px;
+    border: none;
+    border-radius: 2px;
+    margin-right: 10px;
+}
+.primary{
+    color: #fff;
+    background-color: #2231AA;
+}
+.primary:hover{
+    background-color: #5360CD
+}
```

Смотри как именовать классы по БЭМ https://ru.bem.info/methodology/quick-start/
Остальные классы-модификаторы именуй аналогично
```suggestion
.button--primary {
    color: #fff;
    background-color: #2231AA;
}
.button--primary:hover{
    background-color: #5360CD
}
```